### ISSUE TITLE
Support for tags on azurerm_subscription resource

### ISSUE THREAD

**Issue by ondrejvaclavu** [hashicorp/terraform-provider-azurerm#11103]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Tags property in azurerm_subscription resource is read only. Should be writable, so that tags can be assigned in Terraform code, as for all other Azure resources.

### New or Affected Resource(s)

* azurerm_subscription

### Potential Terraform Configuration

```hcl

resource "azurerm_subscription" "example" {
  subscription_name = "My Example EA Subscription"
  billing_scope_id  = data.azurerm_billing_enrollment_account_scope.example.id

  tags = {
    "CostCenter" = "021 - Finance"
    "OwnerEmail" = "finance@company.xyz"
  }
}

```

### References

* https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subscription#example-usage---creating-a-new-alias-and-subscription-for-an-enrollment-account
* https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources

**Comment by Burbert** [comment]
Is there a timeline for this feature? Maybe in the near future? :)

**Comment by jhisc** [comment]
> Is there a timeline for this feature? Maybe in the near future? :)

This PR has a little more information about progress:
https://github.com/hashicorp/terraform-provider-azurerm/pull/12287

**Comment by artur-carvalho** [comment]
Any idea of when this will be available?

**Comment by oswalya** [comment]
This is also a feature I am looking forward to have. Would be great to have this in the provider soon!

**Comment by jackofallops** [comment]
This was closed by #14445 and has been released in provider version 2.88.0.
